compiler/src/parsing/parser.messages

```diff
@@ -5933,7 +5927,7 @@ program: WASMI64 COLON LPAREN UIDENT COMMA WHILE
 ## typ COMMA
 ##
 
-Expected a type or an immediate `)` to complete the tuple type.
```

Comment:
I had missed this. How should I make sure I didn't miss others that maybe said "or an immediate `)`"?

Reply:
There isn't really a great way; you can compile a list of all of the error messages and skim them.

Reply:
I searched for a few phrases that seemed comma, like "comma separated" and "tuple type" and they all still make sense.